src/statmodels.jl

```diff
@@ -224,7 +7,7 @@ Coefficient of determination (R-squared).
 For a linear model, the R² is defined as ``ESS/TSS``, with ``ESS`` the explained sum of squares
 and ``TSS`` the total sum of squares.
 """
-function r2(model::StatisticalModel)
+function StatsAPI.r2(model::StatisticalModel)
```

Comment:
Not needed since you `import`ed it.
```suggestion
function r2(model::StatisticalModel)
```

Reply:
I normally add these for clarity, not because you have to. It just makes it clear the method is not defined here.

Reply:
We don't do this in the rest of StatsBase (e.g. for functions imported from Statistics), so better be consistent.